docs/rpc.md

```diff
@@ -238,6 +238,19 @@ block. It can be removed in future versions, but at the moment you can use it
 to see how much GAS is burned with a particular block (because system fees are
 burned).
 
+#### `getrawnotarypool` and `getrawnotarytransaction` calls
```

Comment:
Make a new section named `P2PNotary extensions`. Add the common comment about P2P Notary enabled networks to the start of the section. Create three subsections: `getrawnotarypool`, `getrawnotarytransaction`, `submitnotaryrequest`. Move the `submitnotaryrequest` under the last one.

Reply:
Done.